src/network/tarcap/packets_handlers/new_dag_block_packet_handler.cpp

```diff
@@ -65,7 +67,7 @@ void DagPacketsHandler::process(const PacketData &packet_data, const std::shared
   onNewBlockReceived(block, new_transactions);
 }
 
-void DagPacketsHandler::sendBlock(dev::p2p::NodeID const &peer_id, taraxa::DagBlock block) {
+void NewDagBlockPacketHandler::sendBlock(dev::p2p::NodeID const &peer_id, taraxa::DagBlock block) {
```

Comment:
because it is packet handler for `NewDagBlockPacket`, the original class name `DagPacketsHandler` was there because it processed multiple dag-related packets but it was changed...

Reply:
but isn't it better to keep names shorter? I do not see any additional value in adding prefix new

Reply:
no it is not. I see the value in it as class names as well as file names are equivalent to the packet names, which makes it easier to orientate in files. If the names would not be the same you would have to check in code which packet handler is registered for which packet type.... IT IS JUST BETTER THIS WAY ! :)

Reply:
so way not call the class and the file same, shorter name -> `dag_block_packet_handler.cpp` and `DagBlockPacketHandler`

Reply:
please just leave it as it is....